### ISSUE TITLE
/xpbar crashes due to new WoW configuration system 

### ISSUE THREAD

**Issue by SpareSimian** [RainmakerUA/XPMultiBar#8]
```
Message: Interface/AddOns/XPMultiBar/XPMultiBar_Config.lua:1820: attempt to call upvalue 'InterfaceOptionsFrame_OpenToCategory' (a nil value)
Time: Wed Aug 14 09:57:40 2024
Count: 1
Stack: Interface/AddOns/XPMultiBar/XPMultiBar_Config.lua:1820: attempt to call upvalue 'InterfaceOptionsFrame_OpenToCategory' (a nil value)
[string "@Interface/AddOns/XPMultiBar/XPMultiBar_Config.lua"]:1820: in function <Interface/AddOns/XPMultiBar/XPMultiBar_Config.lua:1813>
[string "@Interface/AddOns/XPMultiBar/XPMultiBar_Config.lua"]:1827: in function `?'
[string "@Interface/AddOns/Blizzard_ChatFrameBase/Mainline/ChatFrame.lua"]:5468: in function <...AddOns/Blizzard_ChatFrameBase/Mainline/ChatFrame.lua:5414>
[string "=[C]"]: in function `ChatEdit_ParseText'
[string "@Interface/AddOns/Blizzard_ChatFrameBase/Mainline/ChatFrame.lua"]:5120: in function <...AddOns/Blizzard_ChatFrameBase/Mainline/ChatFrame.lua:5119>
[string "=[C]"]: in function `ChatEdit_SendText'
[string "@Interface/AddOns/Blizzard_ChatFrameBase/Mainline/ChatFrame.lua"]:5156: in function `ChatEdit_OnEnterPressed'
[string "*ChatFrame.xml:140_OnEnterPressed"]:1: in function <[string "*ChatFrame.xml:140_OnEnterPressed"]:1>

Locals: (*temporary) = nil
(*temporary) = "Bars"
(*temporary) = "attempt to call upvalue 'InterfaceOptionsFrame_OpenToCategory' (a nil value)"
InterfaceOptionsFrame_OpenToCategory = nil
```

**Comment by SpareSimian** [comment]
The following seems to fix it:
```
--- t:\XPMultiBar/XPMultiBar_Config.lua	2024-08-14 13:06:04.000000000 -0700
+++ XPMultiBar/XPMultiBar_Config.lua	2024-08-14 12:10:27.858433200 -0700
@@ -50,7 +50,6 @@
 local GetAddOnMetadata = GetAddOnMetadata or C_AddOns.GetAddOnMetadata
 local GetLocale = GetLocale
 local GetRestrictedAccountData = GetRestrictedAccountData
-local InterfaceOptionsFrame_OpenToCategory = InterfaceOptionsFrame_OpenToCategory
 local IsXPUserDisabled = IsXPUserDisabled or Utils.EmptyFn
 local UnitLevel = UnitLevel
 
@@ -1817,8 +1816,7 @@
 		for addon root category to be selected in Options dialog.
 		UPD: It works when opening General after Bars.
 	]]
-	InterfaceOptionsFrame_OpenToCategory(L["Bars"])
-	InterfaceOptionsFrame_OpenToCategory(L["General"])
+   Settings.OpenToCategory(md.title)
 end
 
 local function HandleSettingsCommand(input)
```

**Comment by RainmakerUA** [comment]
Hi.
This is already fixed in 1a39611627eb7e1a01d6be76071d7f43e502b4b9
It is time to merge it to master and publish final version.
